src/personalAccessTokens/generateTokenModal/GenerateTokenModal.component.jsx

```diff
@@ -88,6 +88,13 @@ const getAllowedReferersAttribute = ({ allowedReferrers }) => {
     }
 }
 
+const getImportError = ({ report }) => {
+    if (report.errorCode === 'E5003' && report?.errorProperty === 'code') {
```

Comment:
I might have a different view on this than @kabaros 😄, but I feel like it might be better not to do this much mapping of the backend error response on the front end. It feels like this expects us to know a lot about the backend response, and I think it should be up to the backend to provide a readable, comprehensible response rather than having the front end interpret the message 🤔

Reply:
yeah I agree on the general principle - I don't whether we should have this "workaround" for now, since it's quite confusing now when it fails, or wait for a backend fix

The issue also with the backend fix is that replacing `code` with `name` there, will also be fiddly (since this relies on validation done on codes in general in all objects), so there might be some mapping needed anyhow